### DISCUSSION TITLE
Addition of some webshops

### DISCUSSION THREAD

**Discussion by xkuza** [jef/streetmerchant#1816] [Ideas and feature requests]
Hey,

I wanted to ask adding:

rueducommerce.fr
cdiscount.com
grosbill.com
ctronics-computer.de

thanks

**Comment by persohygtfrd** [comment]
hey,
yes me too I woul'd like cdiscount.fr and fnac.fr

**Comment by silkyclouds** [comment]
I was working on a rueducommerce.js file. added the requested lines to the index.js, but then I get into an error when compiling. 

npm ERR! command failed
npm ERR! command sh -c tsc

Can anybody help ?
